ignite/distributed/comp_models/base.py

```diff
@@ -275,8 +276,22 @@ def _do_all_gather(self, tensor: torch.Tensor) -> torch.Tensor:
     def _do_broadcast(self, tensor: torch.Tensor, src: int) -> torch.Tensor:
         pass
 
+    def _check_signature(self, fn: Callable, *args: Any, **kwargs: Any) -> None:
```

Comment:
@fco-dv why we need to check the signature ?
Let's keep it simple and user's responsibility to pass correct args/kwargs to appropriate backend

Reply:
Ok perfect, let's do that it would be simpler indeed